### ISSUE TITLE
Bug: Thumbnails stuck in the internal memory

### ISSUE THREAD

**Issue by Vladkeps** [d4rken-org/sdmaid#5654]
### What happened?
After cleaning thumbnails of 11gb memory was not appear. It's available still 900mb instead of 12 gb. Thumbnails appeared when I copied photos from internal to external sd memory. Funny thing is photos were moved, created thumbnails that simple file manager could not see, and could not be deleted. 

### What did you expect to happen?
I expect to remove thumbnails from internal storage 

### How can I reproduce the issue?
I have no idea how to do this, you guys are experts in this deal. 

### Comments
Could you please advice me how to solve this issue? 
### SD Maid
Version: `5.3.23 (50323)`
Pro: `isPro=false || isComplete=false (unl=false, iap=null, acc=false)`

### Device
Root: `RootContext(rootState=Root(state=UNAVAILABLE)`
Fingerprint: `Sony/G8441/G8441:9/47.2.A.11.228/3311891731:user/release-keys`

**Comment by dragonGod97677** [comment]
Uh can you explain your problem clearly so i can help you out

**Comment by Vladkeps** [comment]
Yas I can!
I had photos on internal memory, one day I decide to cut it all and move to sd card. I did it, but after my memory did not clear (when you cut something and move, memory become free on that space which you move, it's logical) so I tried to figure out why space was not frees on my internal memory. No any of other app did not showed me photos or other files which fill my memory but in same time, memory was almost full and all apps, files, downloads had 9gb of internal space of available 22gb. But another 13 gb was blocked by something, what I could not discover. After installation your app, it was found that 9gb of memory was filled by my apps, files and etc, and another 13 gb was blocked by thumbnails. Of course I delete it rapidly, but this 13 gb didn't become available anymore, after analysing memory by your app it shows me that I have only 900mbs of free space.
Any idea? I hope, I wrote it clear for understanding :))
Thank you guys!

**Comment by d4rken** [comment]
I'm not sure what kind of action is expected here.
If the files are gone, then SD Maid worked correctly. SD Maid doesn't use any kind of "recycle bin", deleted items are gone.
This not freeing storage space is up to your ROM over which i have no control.

Depending on how you determined that the thumbnails are 13GB, the thumbnails itself could have been much smaller if they were sparse files:
https://en.wikipedia.org/wiki/Sparse_file

In that case "site of the thumbnails" could have been 13GB, while "size on disk" could have been 13MB.

> After installation your app, it was found that 9gb of memory was filled by my apps, files and etc, and another 13 gb was blocked by thumbnails.

Where was this shown, what tool in SD Maid, which paths?
Is this reproducible?

**Comment by Vladkeps** [comment]
It was shown in system cleaner menu, and that thumbnails appears when I flag all items in submenu "generic". More than it I saw when thumbnails deleting that name started from ". Thumbnail..." and size was huge.
If the app has logs, I could send it wherever it need.
And. I'm not blaming you guys, I'm trying to figure out what happening with my internal memory and I would like to share my case with you. I think it could be interesting for you.
Hah, just now found another issue. I had changed for Russian language for a moment and when I changed it in English back, submenues are still in Russian. :)
Thanks in advance!
